package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "root",
+  "private": true,
+  "devDependencies": {
+    "husky": "^4.2.5",
+    "lerna": "^3.22.1"
+  },
+  "scripts": {
+    "bootstrap": "lerna bootstrap",
```

Comment:
When is this script used?

Reply:
Bootstrap the packages in the current Lerna repo. Installing all their dependencies and linking any cross-dependencies.